qutebrowser/browser/downloads.py

```diff
@@ -1094,7 +1097,8 @@ def download_delete(self, count=0):
 
     @cmdutils.register(instance='download-model', scope='window', maxsplit=0)
     @cmdutils.argument('count', value=cmdutils.Value.count)
-    def download_open(self, cmdline: str = None, count: int = 0) -> None:
+    @cmdutils.argument('dir', flag='d')
+    def download_open(self, cmdline: str = None, count: int = 0, dir: bool = False) -> None:
```

Comment:
```suggestion
    def download_open(self, cmdline: str = None, count: int = 0, dir_: bool = False) -> None:
```

To avoid shadowing the `dir` builtin. qutebrowser will automatically strip it off when translating it to a command argument.

Also, please break the line so that it's <79 characters.